ecdsa/keys.py

```diff
@@ -78,6 +80,32 @@ def from_der(klass, string):
         assert point_str.startswith(b("\x00\x04"))
         return klass.from_string(point_str[2:], curve)
 
+    @classmethod
+    def from_sec(klass, string, curve=NIST192p, hashfunc=sha1,
+                 validate_point=True):
+        """Convert a public key in SEC binary format to a verifying key."""
+        # based on code from https://github.com/richardkiss/pycoin
+        if string.startswith(b('\x04')):
+            # uncompressed
+            return klass.from_string(string[1:], curve, hashfunc,
+                                     validate_point)
+        elif string.startswith(b('\x02')) or string.startswith(b('\x03')):
+            # compressed
+            is_even = string.startswith(b('\x02'))
+            x = string_to_number(string[1:])
+            order = curve.order
+            p = curve.curve.p()
+            alpha = (pow(x, 3, p) + (curve.curve.a() * x) + curve.curve.b()) % p
+            beta = square_root_mod_prime(alpha, p)
+            if is_even == bool(beta & 1):
+                y = p - beta
+            else:
+                y = beta
+            if validate_point:
+                assert ecdsa.point_is_valid(curve.generator, x, y)
```

Comment:
I made a small test using:
```
print "a"
if True:
    assert False
print "b"
```
and when I run that with -O it prints "a" and "b" showing that it "pass"es, while a non-optimised run will throw an AssertionError after printing "a" (as expected in this example).

So I actually think there is no need to change this PR, nor the current https://github.com/warner/python-ecdsa/blob/master/src/ecdsa/keys.py#L49

@tomato42 and @techguy613 can you confirm that there is no need to change the PR, or clarify why it should and how?